Game/Config/DefaultSpatialGDKEditorSettings.ini

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Please don't modify this file - just keep it modified locally

Reply:
Rookie error ! Will fix this and above 🤕